vmdb/spec/views/host/_form.html.haml_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "spec_helper"
+include ApplicationHelper
```

Comment:
@jrafanie it's needed by rendered view inside that partial, otherwise the test fails with undefined method